perception/navigator_lidar_oa/src/ConnectedComponents.h

```diff
@@ -182,10 +185,9 @@ std::vector< std::vector<int> > ConnectedComponents(OccupancyGrid &ogrid, std::v
 	}
 
 	//Re-organize obstacles, how many connections to count as an obstacle?
-	int newId = 0;
 	objects.clear();
 	for (auto ii : mapObjects)  {
-		if (ii.second.strikesPersist.size() >= 10) {
```

Comment:
Hmm.... I didn't actively change any file modes on my end, so is this a result of the git merge moving files? In any case, easy to fix and update the PR!